ioos_metrics/ioos_metrics.py

```diff
@@ -554,6 +554,58 @@ def hf_radar_installations():
     # This is a hardcoded number at the moment!
     return 165
 
+def mbon_stats():
```

Comment:
You probably want to cache the results for multiple sequencial requests.

```suggestion
@functools.lru_cache(maxsize=128)
def mbon_stats():
```